src/alda/worker.clj

```diff
@@ -45,27 +46,36 @@
 (def current-error (atom nil))
 
 (defn handle-code-play
-  [code]
+  [code history]
   (future
     (reset! current-status :parsing)
     (log/debug "Requiring alda.lisp...")
     (require '[alda.lisp :refer :all])
-    (let [score (try
-                  (log/debug "Parsing input...")
-                  (parse-input code :map)
-                  (catch Throwable e
-                    {:error e}))]
-      (if-let [error (:error score)]
+    (let* [score (try
```

Comment:
ooh thats nice! I'm really used to common lisp/emacs lisp where that isn't the case.